docs/Projects/TAD/reporting_standard.md

```diff
@@ -0,0 +1,312 @@
+            For each result the following fields are present.
+                1. `name` (REQUIRED, string). The name of bar.
+                2. `value` (REQUIRED, float). The value of the corresponding bar.
+
+        2. `graph_plots` (list). The purpose of this field is to capture graph plot like measurements, such as partial dependence
+        plots.
+        There can be multiple graph plots. For each graph plot the following fields are present.
+
+            1. `type` (REQUIRED, string). The type of the graph plot, for example "partial_dependence".
+            2. `name` (OPTONAL, string). A pretty name of the graph, for example "Partial Dependence Plot".
+            3. `results` (list). Results contains the graph plot data. Each graph can depend on a specific output
+            class and feature. There can be multiple results. For each result the following fields are present.
+                1. `class` (OPTIONAL, string/int/float/bool). The output class name that the graph corresponds to.
+                This field is not always present.
+                2. `feature` (REQUIRED, string). The feature the graph corresponds to. This is required, since all
+                relevant graphs are dependend on features.
+                3. `data` (list)
+                    1. `x_value` (REQUIRED, float). The $x$-value of the graph.
+                    2. `y_value` (REQUIRED, float). The $y$-value of the graph.
+
+## Example
+
+### System Card
+
+```yaml
+upl: {upl_uri}                                          # Optional. Example: https://standaarden.overheid.nl/owms/terms/AangifteVertrekBuitenland
+owners:
+- organization: {organization_name}                     # Required. Example: BZK
+  oin: {oin}                                            # Optional. Example: 00000001003214345000
+  name: {owner_name}                                    # Optional. Example: John Doe
+  email: {owner_email}                                  # Optional. Example: johndoe@email.com
+  role: {owner_role}                                    # Optional. Example: Data Scientist.
+
+# Model Cards
+models:
+ - !include {model_card_uri}                            # Optional. Example: model_card.yaml.
+
+# Assessments like IAMA.
+assessments:
+- name: {assessment_name}                               # Required. Example: IAMA.
+  date: {assessment_date}                               # Required. Example: 25-03-2025.
+  contents:
+    - question: {question_text}                         # Required. Example: "Question 1: ...".
+      answer: {answer_text}                             # Required. Example: "Answer: ...".
+      remarks: {remarks_text}                           # Optional. Example: "Remarks: ...".
+      authors:                                          # Optional. Example: "['John', 'Peter']".
+        - name: {author_name}
+      timestamp: {timestamp}                            # Optional. Example: 1711630721.
+```
+
+### Model Card
```

Comment:
In the category "how far do we want to go for the first itterarion?". We could add a section `system_info` that stores information about the system used. F.e. nr CPUs, GPUs, etc.

Reply:
Agreed to not include this at this point.